apps/src/templates/curriculumCatalog/CurriculumCatalog.jsx

```diff
@@ -309,6 +322,20 @@ const CurriculumCatalog = ({
         backgroundUrl={CourseCatalogBannerBackground}
         imageUrl={CourseCatalogIllustration01}
       />
+      {showAssignSuccessMessage && (
+        <div className={style.assignSuccessMessageContainer}>
+          <BodyOneText className={style.assignSuccessMessage}>
+            {assignSuccessMessage}
+          </BodyOneText>
+          <button
+            aria-label="close success message"
+            onClick={() => setShowAssignSuccessMessage(false)}
```

Comment:
Should this also do setAssignSuccessMessage('') to reset that as well? (I don't think this is urgent / worthy of a drone re-run)

Reply:
Good call! I'll add that as a note in the test ticket as part of the cleanup!